docs/examples/LayoutDisplayGrid/SampleLayoutSchema.js

```diff
@@ -0,0 +1,38 @@
+import Ajv from 'ajv';
+import { JSONSchemaBridge } from 'uniforms-bridge-json-schema';
+
+const ajv = new Ajv({ allErrors: true, useDefaults: true });
+
+const schema = {
+  title: 'Guest',
+  type: 'object',
+  properties: {
+    firstName: { type: 'string' },
+    lastName: { type: 'string' },
+    workExperience: {
+      description: 'Work experience in years',
+      type: 'integer',
+      minimum: 0,
+      maximum: 100
+    }
+  },
+  required: ['firstName', 'lastName']
+};
+
+function createValidator(schema) {
+  const validator = ajv.compile(schema);
+
+  return model => {
+    validator(model);
+
+    if (validator.errors && validator.errors.length) {
+      throw { details: validator.errors };
+    }
+  };
+}
+
+const schemaValidator = createValidator(schema);
+
+const bridge = new JSONSchemaBridge(schema, schemaValidator);
+
+export default bridge;
```

Comment:
It can be shortened to `export default new JSONSchemaBridge(schema, schemaValidator);`, like in [SignUpSchema.js](https://github.com/vazco/uniforms/blob/master/docs/examples/CommonForms/SignUpSchema.js#L52)